bin/fuel-core/src/cli/run.rs

```diff
@@ -165,10 +165,22 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
+    /// The starting gas price for the network
+    #[arg(long = "starting-gas-price", default_value = "1000", env)]
+    pub starting_gas_price: u64,
+
+    /// The percentage change in gas price per block
+    #[arg(long = "gas-price-change-percent", default_value = "10", env)]
+    pub gas_price_change_percent: u64,
+
     /// The minimum allowed gas price
-    #[arg(long = "min-gas-price", default_value = "0", env)]
+    #[arg(long = "min-gas-price", default_value = "1000", env)]
```

Comment:
Maybe our default behavior still should be to use a zero gas price to not break tooling and SDKs.  The same for the percent. @Voxelot What do you think?

Reply:
Agree, the CLI defaults should make it easy as possible to start developing locally for the devex onboarding. Production settings should be configured on a per env basis, with the default catering to devs getting started.

Reply:
We've changed everything to `0` for now. Works for me.